src/components/Toast.jsx

```diff
@@ -37,17 +37,19 @@ export class Toast extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     errorMsg: PropTypes.string,
+    status: PropTypes.string,
   };
 
   render() {
-    const { classes, errorMsg } = this.props;
+    const { classes, errorMsg, status } = this.props;
     return (
       <Card className={classes.card}>
         <CardContent className={classes.content}>
           <div className={classes.cardBody}>
             <ErrorIcon className={classes.icon} alt="logo" />
             <div className={classes.text}>
-              {errorMsg || 'An error occurred, please try again or submit a bug report'}
+              {`Error ${status}: ${errorMsg}` ||
+                'An error occurred, please try again or submit a bug report'}
```

Comment:
I'm not sure this is right.  I think the || will never resolve since it's happening outside of the string.  I think you need to include it inside of an expression within the string template